x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/data/DocVector.java

```diff
@@ -243,6 +258,14 @@ public void allowPassingToDifferentDriver() {
 
     @Override
     public void closeInternal() {
-        Releasables.closeExpectNoException(shards.asBlock(), segments.asBlock(), docs.asBlock()); // Ugh! we always close blocks
+        Releasables.closeExpectNoException(
```

Comment:
Can you also call `super.allowPassingToDifferentDriver()` in `allowPassingToDifferentDriver` as we now use the breaker locally here.

Reply:
Ah! I'll poke at it.